### PR TITLE
Split NPM package to two: bundled and installer

### PR DESCRIPTION
**Pull request by Envek**
TLDR: Instead of `@arkweid/lefthook` that ships pre-bundled binaries for all OS and architectures, there will be two: `@evilmartians/lefthook` (the same) and `@evilmartians/lefthook-installer` that will download only needed binary on install (implemented in #188 by @aminya).

A bit of context:

While it is a pain to download megabytes of extra binaries that aren't going to be used at all, installation hooks in NPM packages has some drawbacks also (concerns were raised [here](https://github.com/evilmartians/lefthook/pull/43#issuecomment-591440728) and [here](https://github.com/evilmartians/lefthook/pull/188#issuecomment-860356225)), including security concerns (and there were a lot of examples last months, unfortunately). Because of this I decided to publish both variants.

 - Installer variant is already in `master` branch (implemented in #188)
 - Bundled variant is restored from `0-7-stable` branch

### CODE REVIEW

File: docs/full_guide.md [hunk 1]
```diff
@@ -10,18 +10,30 @@ go get github.com/evilmartians/lefthook
...
+
+ 1. [@evilmartians/lefthook](https://www.npmjs.com/package/@evilmartians/lefthook) with pre-bundled binaries for all architectures:
+
+    ```bash
+    npm install @evilmartians/lefthook --save-dev
+    # or yarn:
+    yarn add -D @evilmartians/lefthook
+    ```
+
+ 2. [@evilmartians/lefthook-installer](https://www.npmjs.com/package/@evilmartians/lefthook) that wil fetch binary file on installation:
```
Comment (mrexox) [inline docs/full_guide.md:23]: The link is the same as in 1. Should it be different?

```suggestion
 2. [@evilmartians/lefthook-installer](https://www.npmjs.com/package/@evilmartians/lefthook-installer) that wil fetch binary file on installation:
```

File: cmd/templates/hook.tmpl [hunk 2]
```diff
@@ -17,18 +17,21 @@ call_lefthook()
   if lefthook{{.Extension}} -h >/dev/null 2>&1
   then
     eval lefthook{{.Extension}} $@
-  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/lefthook_${osArch}_${cpuArch}/lefthook{{.Extension}}"
+  elif test -f "$dir/node_modules/@evilmartians/lefthook/bin/lefthook_${osArch}_${cpuArch}/lefthook{{.Extension}}"
   then
-    eval "$dir/node_modules/@arkweid/lefthook/bin/lefthook_${osArch}_${cpuArch}/lefthook{{.Extension}} $@"
+    eval "$dir/node_modules/@evilmartians/lefthook/bin/lefthook_${osArch}_${cpuArch}/lefthook{{.Extension}} $@"
+  elif test -f "$dir/node_modules/@evilmartians-installer/lefthook/bin/lefthook_${osArch}_${cpuArch}/lefthook{{.Extension}}"
```
Comment (mrexox) [inline cmd/templates/hook.tmpl:23]: I guess there is a typo:
```suggestion
  elif test -f "$dir/node_modules/@evilmartians/lefthook-installer/bin/lefthook_${osArch}_${cpuArch}/lefthook{{.Extension}}"
```

**Review by mrexox — COMMENTED** [review]
Just a few typos, but overall I like it :cake:

**Review by mrexox — APPROVED** [review]
